storage_v2/v1adapter/translator.go

```diff
@@ -61,6 +61,23 @@ func V1TracesFromSeq2(otelSeq iter.Seq2[[]ptrace.Traces, error]) ([]*model.Trace
 	return jaegerTraces, nil
 }
 
+// Returns
+func V1TraceToOtelTrace(jTrace model.Trace) *ptrace.Traces {
```

Comment:
we always pass model.trace as pointer in the rest of the code.
```suggestion
func V1TraceToOtelTrace(jTrace *model.Trace) *ptrace.Traces {
```